Yoast/wordpress-seo
# Load translations on api call

## Summary

This PR can be summarized in the following changelog entry:

## Relevant technical choices:

* Add object for loading the translations in the service instead of `localize_script`

## Test instructions

This PR can be tested by following these steps:

* Checkout trunk.
* In the onboarding wizard go to step 2 and see the navigation buttons at the bottom in English.
* Checkout this branch. 
* In class `WPSEO_Configuration_Service` change the line `$this->set_translations( new WPSEO_Configuration_Translations( WPSEO_Utils::get_user_locale() ) );` into `$this->set_translations( new WPSEO_Configuration_Translations( 'nl_NL' ) );`.
* Go to step 2 and see the values of the buttons being translated.
